src/Common.OData.ApiExplorer/AspNet.OData/DefaultModelTypeBuilder.cs

```diff
@@ -66,6 +67,14 @@ public Type NewStructuredType( IEdmStructuredType structuredType, Type clrType,
                 var propertyType = property.PropertyType;
                 var structuredTypeRef = structuralProperty.Type;
 
+                if ( passedTypes.TryGetValue( structuredTypeRef.ToString(), out var passedType ) )
+                {
+                    properties.Add(new ClassProperty(property, passedType));
```

Comment:
nit: the spacing is off by my rules, but this should be flagged by code analysis and auto-formatted by the .editorconfig settings. correcting this should be as easy as formatting the document.

>Definitely let me know you don't see CA warnings and/or formatting the document doesn't fix all of these. You're not supposed to have to guess or do a lot of work to format things the way _I want_ them to be. ;)

Reply:
It formats for methods, but I had to manually add the formatting rules for nameof, typeof etc.
I'm using Jetbrains Rider if that means anything.